src/components/transaction_list/index.tsx

```diff
@@ -1,7 +1,71 @@
 import React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+import { TransactionEntry } from "../../types";
 
-const TransactionList: React.FC = () => {
-  return <h2>Transaction List Component</h2>;
+interface TransactionEntries {
+  entries: Array<TransactionEntry>;
+}
+
+const NoResults: React.FC = () => (
+  <TableRow>
+    <TableCell>Empty</TableCell>
+    <TableCell>Empty</TableCell>
+    <TableCell>Empty</TableCell>
+  </TableRow>
+);
+
+const Transaction = ({ id, date, description, amount }: TransactionEntry) => {
+  return (
+    <TableRow key={id}>
+      <TableCell>{date.toDateString()}</TableCell>
+      <TableCell>{description}</TableCell>
+      <TableCell>{amount}</TableCell>
+    </TableRow>
+  );
+};
+
+const TransactionList: React.FC<TransactionEntries> = ({ entries }) => {
+  console.log("entries", entries);
+
+  return (
+    <TableContainer>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Date</TableCell>
+            <TableCell>Description</TableCell>
+            <TableCell>Amount($)</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {/* TODO
+            - RENDER `NoResults` when entries.length === 0 otherwise render the transactions
+            - Also there are warning issues about each child in the list should have a unique key
+              So i tried adding id for the table rows in the transaction component, but doesn't fix
+          */}
+          {entries.map((entry) => {
+            const { id, date, description, amount } = entry;
+            console.log(date.toDateString(), description, amount);
+            return (
+              <Transaction
+                id={id}
+                date={date}
+                description={description}
+                amount={amount}
+              />
+            );
+          })}
```

Comment:
## TODO
- Fix conditional rendering
   -  Check the if `entries.length === 0` then render the `NoResults` component
- Fix warnings given from console about unique keys